includes/filters.php

```diff
@@ -117,13 +129,36 @@ function give_akismet( $spam ) {
 	$ignore = array( 'HTTP_COOKIE', 'HTTP_COOKIE2', 'PHP_AUTH_PW' );
 
 	foreach ( $_SERVER as $key => $value ) {
-		if ( ! in_array( $key, (array) $ignore ) ) {
-			$args["$key"] = $value;
+		if ( ! in_array( $key, $ignore, true ) ) {
+			$args[$key] = $value;
 		}
 	}
 
+	$response = give_akismet_spam_check_post( $args );
+	$spam     = 'true' === $response[1];
+
+	// Log spam information.
+	if( $spam ) {
+		give_record_log(
+			sprintf(
+				'Spam flagged for %1$s (%2$s)',
```

Comment:
Change to "This donor's email (John Doe - johndoe@gmail.com) has been flagged as SPAM. Click here to whitelist this email if you feel it was flagged incorrectly." 

- Open link in new tab.